docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -62,6 +62,10 @@ In the "SSL" tab, set "SSL Mode" to `Verify-Full`:
 
 Click "Save", and pgAdmin should immediately connect.
 
+<Admonition type="note">
+  In order to use a service profile on a Windows client you will first have to export a PG Service File environmental variable in the terminal with `setx PGSERVICEFILE <path_to_.pg_service.conf_file>` and then restart the pgAdmin client. The `.pg_service.conf` file will be created in the user's home directory after a user logs into a database via `tsh db login <dbname>`.
```

Comment:
You can make this shorter without loosing meaning:

On a Windows client, use terminal to export a PG Service File env variable:
 ```bash
  $ setx PGSERVICEFILE <path_to_.pg_service.conf_file>
  ```
 Restart the pgAdmin client.